src/js/DrawStyleController.js

```diff
@@ -188,9 +213,20 @@ goog.require('ga_styles_service');
     $scope.options.color = options.colors[5];
     $scope.options.textColor = options.colors[5];
     $scope.options.textSize = options.textSizes[0];
-    $scope.options.icon = options.icons[0];
     $scope.options.iconColor = options.colors[5];
     $scope.options.iconSize = options.iconSizes[2];
+    $scope.options.iconCategory = options.iconCategories[0];
+
+    $scope.options.iconCategories.forEach(function(category) {
+      if (category.type === 'img') {
+        category.icons = getImgsByCategory(category);
+      } else {
+        category.icons = iconsCSS;
```

Comment:
You could potentially have more than one category with css icons . So remove this `else` put the icons directly in the category's definition.
```
{id: 'standard', icons: iconsCSS ...
```